app/assets/javascripts/ama_layout/desktop/sticky-footer.coffee

```diff
@@ -1,6 +1,20 @@
-$(window).bind "load", ->
+root = exports ? this
+
+root.adjustFooter = ->
```

Comment:
Do these functions need to be defined globally? Could they just be defined and called within the window.load event handler?